### DISCUSSION TITLE
github page turning bug

### DISCUSSION THREAD

**Discussion by mq00fc** [community/community#61207] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

![image](https://github.com/community/community/assets/50408721/f80a737d-2ef6-40ae-99d0-fd05142bf6bc)
If you use Google Translate or Edge Translate to turn the page, there will be an overview that triggers the above error, and you can continue to use it when you refresh, but there will still be a probability of triggering when you continue to turn the page

The browsers used are chrome and edge (latest version)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @mq00fc ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
